calculator.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
```

Comment:
Awesome job! You followed all the guidelines perfectly! 💯 Very happy to see the improvement from your last assignment! 😃